README.md

```diff
@@ -32,4 +32,10 @@ sudo apt -y install docker.io docker-compose
 sudo groupadd docker
 sudo usermod -aG docker $USER
 newgrp docker
-```
\ No newline at end of file
+```
+
+## Source code structure
+
+Please refer to [this](https://meaningfy.atlassian.net/l/c/bK0uVdG7) page.
```

Comment:
if we're planning to have this open-source, then this info would not be accessible to GitHub repo readers.